R/pkg/R/SQLContext.R

```diff
@@ -840,6 +844,7 @@ createExternalTable <- function(x, ...) {
 #'                      clause expressions used to split the column `partitionColumn` evenly.
 #'                      This defaults to SparkContext.defaultParallelism when unset.
 #' @param predicates a list of conditions in the where clause; each one defines one partition
+#' @param ... additional argument(s) passed to the method.
```

Comment:
in this case I'd reference the same in L829, something like:
" additional JDBC database connection named properties"